### PR TITLE
Blog Page

### PR DESCRIPTION
**Pull request by SaraVieira**
👋

This PR adds a blog page that gets the articles from medium and the local directory

It combines them and orders them to show by date

### CODE REVIEW

File: packages/homepage/gatsby-node.js [hunk 1]
```diff
@@ -49,6 +50,69 @@ exports.createPages = async ({ graphql, actions }) => {
...
+    }
+  `);
+
+  blogsFromMedium.data.allFeedMediumBlog.edges.forEach(edge => {
+    const slug = edge.node.title
+      .toLowerCase()
+      .replace(/[^\w ]+/g, '')
+      .replace(/ +/g, '-');
+    const id = edge.node.id;
+    if (edge.node.categories) {
```
Comment (SaraVieira) [inline packages/homepage/gatsby-node.js]: If no categories its a comment

File: packages/homepage/src/utils/makePosts.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// Im so sorry about this whole file
```
Comment (SaraVieira) [inline packages/homepage/src/utils/makePosts.js]: 🙃

File: packages/homepage/gatsby-config.js [hunk 3]
```diff
@@ -71,10 +71,21 @@ module.exports = {
...
+      },
+    },
     {
       resolve: `gatsby-plugin-google-fonts`,
       options: {
-        fonts: [`Poppins:600,700,800`, `source sans pro:300,400,500,600,700`],
+        fonts: [
+          `Poppins:600,700,800`,
+          `source sans pro:300,400,500,600,700`,
+          'open sans:400',
```
Comment (CompuIves) [inline packages/homepage/gatsby-config.js]: What do we use open sans for?

File: packages/homepage/gatsby-node.js (hunk 1 above)
Comment (CompuIves) [reply packages/homepage/gatsby-node.js]: Would be good to add this as a comment (hah!) so future readers know this too.

File: packages/homepage/gatsby-node.js [hunk 4]
```diff
@@ -49,7 +50,72 @@ exports.createPages = async ({ graphql, actions }) => {
...
+    `
+  );
+
+  if (allMarkdownArticles.data) {
+    allMarkdownArticles.data.allMarkdownRemark.edges.forEach(edge => {
+      const slug = edge.node.frontmatter.slug;
+      const id = edge.node.id;
+
+      createPage({
+        path: 'post/' + slug,
```
Comment (CompuIves) [inline packages/homepage/gatsby-node.js]: Would it make sense to do `/blog/` prefix?

File: packages/homepage/src/components/PostElements.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 1.5;
+
+  color: #f2f2f2;
+`;
+
+export const PostDate = styled.span`
+  font-family: 'Poppins';
```
Comment (CompuIves) [inline packages/homepage/src/components/PostElements.js]: I know that I've been doing this wrong as well, but would be better to have this as a fallback.

```suggestion
  font-family: 'Poppins', sans-serif;
```

File: packages/homepage/src/pages/blog.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  Aside,
+  Header,
+  PageTitle,
+  PageSubtitle,
+} from './_blog.elements';
+
+import Layout from '../components/layout';
+import { makeFeed } from '../utils/makePosts';
+
+// UNCOMMENT AT THE BOTTOM IF IT BREAKS
```
Comment (CompuIves) [inline packages/homepage/src/pages/blog.js]: Will this be uncommented?

File: packages/homepage/src/templates/docs.js [hunk 7]
```diff
@@ -274,7 +274,6 @@ export const pageQuery = graphql`
             title
           }
           fields {
-            slug
```
Comment (CompuIves) [inline packages/homepage/src/templates/docs.js]: Don't we need slug for docs?

File: packages/homepage/src/utils/makePosts.js (hunk 2 above)
Comment (CompuIves) [reply packages/homepage/src/utils/makePosts.js]: I like it!

File: packages/homepage/gatsby-config.js (hunk 3 above)
Comment (SaraVieira) [reply packages/homepage/gatsby-config.js]: The text in the post single page

File: packages/homepage/src/pages/blog.js (hunk 6 above)
Comment (SaraVieira) [reply packages/homepage/src/pages/blog.js]: Only when someone adds a markdown blog post

File: packages/homepage/src/templates/docs.js (hunk 7 above)
Comment (SaraVieira) [reply packages/homepage/src/templates/docs.js]: They don't even have them in the code ahah

**Pull request merged**